src/app/modules/form/form.service.ts

```diff
@@ -343,3 +345,70 @@ export const retrievePublicFormsWithSmsVerification = (
     return okAsync(forms)
   })
 }
+
+export const createSingleSampleSubmissionAnswer = (field: FormFieldDto) => {
+  let sampleValue = null
+  let noOfOptions = 0
+  let randomSelectedOption = 0
+  switch (field.fieldType) {
+    case 'textarea':
+    case 'textfield':
+      sampleValue =
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+      break
+    case 'radiobutton':
+    case 'dropdown':
+      noOfOptions = field.fieldOptions.length
+      randomSelectedOption = Math.floor(Math.random() * noOfOptions)
+      sampleValue = field.fieldOptions[randomSelectedOption]
+      break
+    case 'email':
+      sampleValue = 'hello@example.com'
+      break
+    case 'decimal':
+      sampleValue = 1.234
+      break
+    case 'number':
+      sampleValue = 1234
+      break
+    case 'mobile':
+      sampleValue = '+6598765432'
+      break
+    case 'homeno':
+      sampleValue = '+6567890123'
+      break
+    case 'yes_no':
+      sampleValue = 'yes'
+      break
+    case 'rating':
+      sampleValue = 1
+      break
+    case 'attachment':
+      sampleValue = 'attachmentFileName'
+      break
+    default:
+      break
+  }
+  let answer = {}
+  if (sampleValue != null) {
+    answer = {
+      id: field._id,
+      question: field.title,
+      answer: sampleValue,
+      fieldType: field.fieldType,
+    }
+  }
+  return answer
```

Comment:
It feels wrong to return an empty object if the value is null (empty objects are truthy in JS) 🤔, and `createSampleSubmissionResponses()` below only skips `null`/`undefined`. 

I'd advise to either return `null`, or to return an object which contain the static fields taken from the definition (`id`, `question`, `fieldType`), and leave `answer` to be `undefined` or `null`. In the later case, code can be simplified a little:

```suggestion
  return  {
    id: field._id,
    question: field.title,
    answer: sampleValue,
    fieldType: field.fieldType,
  }
```

Reply:
went with the suggestion! [d982f83](https://github.com/opengovsg/FormSG/pull/6325/commits/d982f831f401f1651d252f02bd99e0e792e8d65d)